### PR TITLE
chore: update logger adapter constructors

### PR DESCRIPTION
**Pull request by christyjacob4**
Previously our constructors accepted only a single string, and parsed it in different ways to extract keys and IDs out of them. This was like magic and now very verbose. 

With this refactor, we introduce explicit parameters in the constructor for each of the required attributes of the adapter.

### CODE REVIEW

File: src/Logger/Adapter/LogOwl.php [hunk 1]
```diff
@@ -24,6 +24,22 @@ class LogOwl extends Adapter
      */
     protected string $logOwlHost;
 
+    /**
+     * LogOwl constructor.
+     *
+     * @param  string  $ticket
+     * @param  string  $host
+     */
+    public function __construct(string $ticket, string $host)
```
Comment (Meldiron) [inline src/Logger/Adapter/LogOwl.php:33]: Host should be optional, `string $host = 'https://api.logowl.io/logging/'`

File: src/Logger/Adapter/Sentry.php [hunk 2]
```diff
@@ -26,47 +26,19 @@ class Sentry extends Adapter
...
     /**
      * Sentry constructor.
      *
-     * @param  string  $dsn
+     * @param  string  $projectId
+     * @param  string  $key
+     * @param  string  $host
      */
-    public function __construct(string $dsn)
+    public function __construct(string $projectId, string $key, string $host)
```
Comment (Meldiron) [inline src/Logger/Adapter/Sentry.php:38]: Same here, we can make host optional with default value

Comment (christyjacob4) [reply src/Logger/Adapter/Sentry.php]: done 👍

File: src/Logger/Adapter/LogOwl.php (hunk 1 above)
Comment (christyjacob4) [reply src/Logger/Adapter/LogOwl.php]: done 👍

**Review by Meldiron — APPROVED** [review]

**Pull request merged**